ops/travis/e2e-testing/demo/puppeteerDemo.js

```diff
@@ -35,13 +35,15 @@ async function run () {
   // NEW STUDY from Template
```

Comment:
I still think that all the e2e-testing machinery should NOT be under ``osparc-simcore/ops/travis`` but rather under something like ``osparc-simcore/tests/end-to-end``. The former shall be SOLELY for travis scripts and the latter must be runnable in place as well (not only via the travis scripts)

Reply:
Done!